tests/plugins/templates/templates.js

```diff
@@ -0,0 +1,26 @@
+/* bender-tags: editor,dialog */
+/* bender-ckeditor-plugins: wysiwygarea,toolbar,basicstyles,templates,format*/
+
+( function() {
+    'use strict';
+
+    bender.test( {
+        'Abort template insertion': function() {
```

Comment:
We have a convention that every test should start with `test`, so it should look like:

```suggestion
        'test aborting template insertion': function() {
```

Please update the test name accordingly.